### ISSUE TITLE
Provide container image for arm64

### ISSUE THREAD

**Issue by jnohlgard** [cockpit-project/cockpit-container#22]
Hi,
I am using Fedora IoT on 64 bit ARM (aarch64). The cockpit packages in dnf are available on aarch64, but it would be nice if the container images also were built for this. Currently x86_64 is the only architecture available for cockpit-ws at Docker Hub.

Docker hub has support for multiple architectures in the same repo, see https://www.docker.com/blog/multi-arch-build-and-images-the-simple-way/
Example of a multi arch repo: https://hub.docker.com/_/influxdb?tab=tags&page=1&ordering=last_updated

**Comment by svenfoo** [comment]
I managed to build 64 bit ARM (aarch64) images based on the Dockerfile etc provided by this repository. But I had to do some modifications:
-  remove the `cockpit-preview` repo as it doesn't seem to have an `aarch64` variant
- add `updates-testing` repos as they have recent cockpit-ws and cockpit-bridge packages for `aarch64
`
The changes are here: https://github.com/svenfoo/cockpit-container/tree/aarch64

**Comment by martinpitt** [comment]
I enabled aarch64 in the cockpit-preview COPR repo, thanks for pointing out!

**Comment by martinpitt** [comment]
Just to make sure, @svenfoo 's changes should not be necessary any more since March 30, our cockpit-preview copr has had arm64 builds since then. I'm not sure yet how to build arm64 on quay.io though, I haven't found any instructions or settings for that.

**Comment by svenfoo** [comment]
Right, I reverted the changes in my copy and have been able to build images for the Raspberry Pi recently.

**Comment by danielvandenberg95** [comment]
Since I keep running into this issue when googling my own issue:
https://github.com/danielvandenberg95/cockpit-ws/pkgs/container/cockpit-ws
cockpit-ws in docker on debian.

**Comment by martinpitt** [comment]
We are going to remove this cockpit-container repository (it was never really meant to be an user-facing one), so transferring to cockpit.
